app/helpers/sessions_helper.rb

```diff
@@ -1,27 +1,11 @@
 module SessionsHelper
-  def log_in(user)
```

Comment:
**Code quality - critical**

Fix misuse of the `SessionsHelper` by removing non-view logic